cmd/helm/pull.go

```diff
@@ -43,7 +43,7 @@ result in an error, and the chart will not be saved locally.
 `
 
 func newPullCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
-	client := action.NewPullWithOpts(action.WithConfig(cfg))
+	client := action.NewPullWithOpts(action.WithConfig(cfg), action.WithPullOptWriter(out))
```

Comment:
no need for `action.WithPullOptWriter`. We don't use it anymore since [11480](https://github.com/helm/helm/pull/11480/files). Everything is redirected to `OS.Stderr`.